--- a/src/lal_tools.gpr
+++ b/src/lal_tools.gpr
@@ -18,7 +18,8 @@
    for Object_Dir use "../obj/" & LAL_Tools_Common.Superproject & "/lib/" & Library_Type;
    for Library_Kind use Library_Type;
    for Library_Dir use "../lib/" & Library_Type;
-   for Library_Name use "lal_tools";
+   for Library_Name use "adalang_tools";
+   for Library_Version use "libadalang_tools.so." & external ("VERSION");
 
    for Languages use ("Ada", "Templates");
    package Naming is
